### PR TITLE
Style CompareModificationsView

### PR DESCRIPTION
**Pull request by lewfish**
In /compare, the list of modifications should now be styled like the wireframe:
https://cloud.githubusercontent.com/assets/5880793/8658465/e667dde8-2971-11e5-82f1-3768dd487d3e.png

Connects #592

### CODE REVIEW

File: src/mmw/js/src/compare/controllers.js [hunk 1]
```diff
@@ -35,8 +35,10 @@ function makeTestProject() {
             'type': '',
             'area':3.361859935949525,
             'units':'km<sup>2</sup>'
-        }],
-        scenarios = new modelingModels.ScenariosCollection(
+        }];
+    modifications = modifications.concat(modifications);
```
Comment (lliss) [inline src/mmw/js/src/compare/controllers.js]: Doubled up on purpose?

Comment (lewfish) [inline src/mmw/js/src/compare/controllers.js]: Yes, I just wanted a bunch of modifications to show off the scrolling.

Comment (mmcfarland) [inline src/mmw/js/src/compare/controllers.js]: Pretty sure this line got doubled-up.

**Comment by mmcfarland** [comment]
@azavea-bot rebuild

**Comment by mmcfarland** [comment]
+1 if the build passes.  All tests and lint passed locally for me.

**Pull request merged**